### PR TITLE
Init contribution

### PR DESCRIPTION
**Pull request by emielvanseveren**
- Add contribution information
- Basic explanation of what Bantr is

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,75 @@
...
+  <br/>
+  <a href="https://bantr.app">Bantr</a>
+    <img src="https://github.com/Bantr/Spawn/blob/master/public/static/assets/icon/4x/color-combo-2%404x.png" align="center" alt="Bantr" width="100"></a>
 </h1>
 
 <h3 align="center">Bantr is free to use. But not to operate. Please consider supporting our service!</h3>
+<p align="center">The repository contains the frontend of the application and is completely written in <strong>React</strong>.</p>
+
+## What is Bantr?
+Originally Bantr started as a [CSGO](https://store.steampowered.com/app/730/CounterStrike_Global_Offensive/) ban notifier. We often found ourselves frustrated playing against cheaters. As a player there is nothing more you can do than report them. It often remained unknown if that player received a ban or just continued cheating or maybe he just had a good day. Ban notifier tracks all the players you played against and will keep track of their ban record. When they receive a ban, Bantr will notify you.
```
Comment (niekcandaele) [inline README.md:14]: Switching Bantr and Ban notifier here, should stick to Bantr

File: README.md [hunk 2]
```diff
@@ -1,13 +1,75 @@
...
+
+To get the data (players) you played against we need to parse the game demos. These demos contains lots more data. The idea grew and we decided to extract that data and create a data analytics platform. 
+
+Bantr is a data analytics platform that processes match data from Counterstrike Global Offensive player's.
```
Comment (niekcandaele) [inline README.md:18]: players

File: README.md [hunk 3]
```diff
@@ -1,13 +1,75 @@
...
+
+## What is Bantr?
+Originally Bantr started as a [CSGO](https://store.steampowered.com/app/730/CounterStrike_Global_Offensive/) ban notifier. We often found ourselves frustrated playing against cheaters. As a player there is nothing more you can do than report them. It often remained unknown if that player received a ban or just continued cheating or maybe he just had a good day. Ban notifier tracks all the players you played against and will keep track of their ban record. When they receive a ban, Bantr will notify you.
+
+To get the data (players) you played against we need to parse the game demos. These demos contains lots more data. The idea grew and we decided to extract that data and create a data analytics platform. 
+
+Bantr is a data analytics platform that processes match data from Counterstrike Global Offensive player's. 
+### Our vision
+Currently we are just working and learning as much as we can, the goal is to create a very extensive, fast, good looking data analytics platform that is free for everyone. We are open source and have no intensions to make profits.
```
Comment (niekcandaele) [inline README.md:20]: good-looking

intentions

File: README.md [hunk 4]
```diff
@@ -1,13 +1,75 @@
...
+
+### Status
+Bantr is currently mainly developed by 2 college students since the beginning of 2020. You can see our current priorities and what we are working on in the main project board. We have recently released the first alpha version and are working hard on a feature rich beta version.
```
Comment (niekcandaele) [inline README.md:23]: Will you update this every release/status change?

File: README.md [hunk 5]
```diff
@@ -1,13 +1,75 @@
...
+
+#### Technologies
```
Comment (niekcandaele) [inline README.md:25]: You could add links to the technologies here if you want

File: README.md [hunk 6]
```diff
@@ -1,13 +1,75 @@
...
+Here is a list of all the largest utilities we use:
+
+- **React** - Frontend React app
+- **Cypress** - Regression tests (mocked)
+- **Storybook** - Isolates components. Easier for creating and testing its behavior.
+
+## Contributing
+We heartily welcome any and all contributions! Feel free to create a pull request and we are looking forward to review your code.
```
Comment (niekcandaele) [inline README.md:33]: "Hop into Discord if you have trouble getting started" or something?

File: README.md [hunk 7]
```diff
@@ -1,13 +1,75 @@
...
+├── cypress       # Cypress mocked tests
+├── lib           # Shared functions & reusable components
+├── public        # files that are served 
+├── scripts       # Scripts 
+├── src           # Frontend SPA
+├── stories       # Global storybook
+```
+
+### First time setup
+The first step to running Bantr locally is downloading the code by cloning the repository:
```
Comment (niekcandaele) [inline README.md:47]: Do we need to have this much details for cloning a repo? I assume if someone wants to contribute code, they know how to clone a repo already right?

File: README.md [hunk 8]
```diff
@@ -1,13 +1,75 @@
...
+```
+ If you get `Permission denied` error using `ssh` refer [here](https://help.github.com/articles/error-permission-denied-publickey/)
+or use `https` link as a fallback.
+```sh
+git clone https://github.com/Bantr/Spawn.git
+```
+
+### Installation
+```sh
+npm install - Install dependencies
```
Comment (niekcandaele) [inline README.md:59]: Would change this to

```sh
npm install # Install dependencies
npm start # Starts the application using webpack's dev server
```

For easier copy-paste

File: README.md [hunk 9]
```diff
@@ -1,13 +1,75 @@
...
+npm start - Starts the application using webpack's dev server
+```
+
+### Secrets
```
Comment (niekcandaele) [inline README.md:63]: API_ROOT?

File: README.md [hunk 10]
```diff
@@ -1,13 +1,75 @@
...
+on the root of the repo you can find an example .env file ([.env.example](https://github.com/Bantr/Spawn/blob/master/.env.example)). Create a new .env file on the root of the repository and add the following 3 variables
+- **HOSTNAME** - The ip/name the webpack-dev-server should start the application (default: localhost)
+- **PORT** - The port the webpack-dev-server should start the application (default: 8080)
+- **HASURA_GRAPHQL_USER_SECRET** - Required to download the get the graphql types (runs before dev-server).
+
+How do you obtain the **HASURA_GRAPHQL_USER_SECRET**? <br/> Go to the current Bantr.app and login with your steam account (create one if you don't have one, they are free). When authenticated open a new tab and surf to https://api.bantr.app/auth/session. This will return your session object. One of the keys will be **'graphQLKey'**. This is the key you need.
+
+### First contribution
+Add your name to the **Contributors.md** file. Don't add it at the beginning or end of the file. Put it anywhere in between.
```
Comment (niekcandaele) [inline README.md:72]: Where is this file?

Whats the purpose of this file? Aren't contributors already logged by Git?

And whats wrong with the beginning and end of the file? :p

File: README.md (hunk 4 above)
Comment (emielvanseveren) [reply README.md:23]: Yes, I'll add it the 'things I should do before every release' list

File: README.md (hunk 7 above)
Comment (emielvanseveren) [reply README.md:47]: True, I'll delete this

File: README.md (hunk 9 above)
Comment (emielvanseveren) [reply README.md:63]: ah yes

File: README.md (hunk 10 above)
Comment (emielvanseveren) [reply README.md:72]: +1

Comment (emielvanseveren) [reply README.md:72]: Should add a contribution guideline tho

File: README.md [hunk 11]
```diff
@@ -42,34 +43,18 @@ Bantr/Spawn
...
+npm start  # Starts the application using webpack's dev server
 ```
 
 ### Secrets
 on the root of the repo you can find an example .env file ([.env.example](https://github.com/Bantr/Spawn/blob/master/.env.example)). Create a new .env file on the root of the repository and add the following 3 variables
-- **HOSTNAME** - The ip/name the webpack-dev-server should start the application (default: localhost)
+- **HOSTNAME** - The address the webpack-dev-server should start the application (default: localhost)
 - **PORT** - The port the webpack-dev-server should start the application (default: 8080)
 - **HASURA_GRAPHQL_USER_SECRET** - Required to download the get the graphql types (runs before dev-server).
+- **API_ROOT** - The address of the api. This should be set to **api.bantr.app*
```
Comment (niekcandaele) [inline README.md:57]: Unless you run the API locally for testing :slightly_smiling_face:

**Pull request merged**